.github/workflows/integration_test.yml

```diff
@@ -80,3 +80,10 @@ jobs:
           name: kubernetes-integration-test-diag
           path: /tmp/ktf-diag*
           if-no-files-found: ignore
+
+      - name: Notify Slack on Test Failure
+        if: failure()
+        uses: rtCamp/action-slack-notify@v2
```

Comment:
It might be better to use the action supported by Slack themselves.
There's a fancy example here: https://github.com/slackapi/slack-github-action/blob/main/example-workflows/Technique_3_Slack_Incoming_Webhook/blocks.yml
And a much simpler one here: https://github.com/slackapi/slack-github-action/blob/main/example-workflows/Technique_3_Slack_Incoming_Webhook/text.yml